viadot/tasks/azure_sql.py

```diff
@@ -277,6 +285,11 @@ def run(
         result = azure_sql.run(query)
 
         self.logger.info(f"Successfully ran the query.")
+        if save_query == True:
+            file = open(file_path, "w")
```

Comment:
Use context manager to open/modify file -> ` with open(filename,'w') as file:...`